backends-velox/src/main/scala/org/apache/spark/sql/execution/unsafe/UnsafeBytesBufferArray.scala

```diff
@@ -48,6 +48,7 @@ case class UnsafeBytesBufferArray(arraySize: Int, bytesBufferLengths: Array[Int]
         "not equal to buffer lengths!")
     assert(totalBytes >= 0, "Unsafe buffer array total bytes can't be negative!")
   }
+  private val allocatedBytes = totalBytes + 7;
```

Comment:
I think it is not totalBytes + 7, it is the mutiples of 8

Reply:
I am not sure I am following? `allocatedBytes` is the number of bytes, not the number of long integers.

Reply:
If better change it in a follow up PR to `totalBytes + 7）/8 * 8`

Reply:
No problem. I am updating this PR.